packages/amplify-category-geo/src/service-utils/deviceLocationTrackingParams.ts

```diff
@@ -0,0 +1,35 @@
+import _ from 'lodash';
+import { printer } from 'amplify-prompts';
+import { ResourceParameters } from './resourceParams';
+
+/**
+ * DeviceLocationTrackingParameters
+ */
+export type DeviceLocationTrackingParameters = ResourceParameters & {
+    groupPermissions: string[];
+    roleAndGroupPermissionsMap: Record<string, string[]>;
+};
+
+/**
+ * isCompleteTrackingParams
+ */
+export const isCompleteTrackingParams = (
+  partial: Partial<DeviceLocationTrackingParameters>,
+): partial is DeviceLocationTrackingParameters => {
+  printer.info(`params: ${JSON.stringify(partial)}`);
+  const requiredFields = ['providerContext', 'name', 'accessType'];
+  const missingField = requiredFields.find(field => !_.keys(partial).includes(field));
```

Comment:
Lodash for this seems like overkill. Any reason not to use `Object.keys`?

Reply:
Agreed, will update